api/eth_transactions.go

```diff
@@ -0,0 +1,508 @@
+package api
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	mathbig "math/big"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/filecoin-project/go-address"
+	gocrypto "github.com/filecoin-project/go-crypto"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	builtintypes "github.com/filecoin-project/go-state-types/builtin"
+	"github.com/filecoin-project/go-state-types/builtin/v10/eam"
+	typescrypto "github.com/filecoin-project/go-state-types/crypto"
+
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+const Eip1559TxType = 2
+
+type EthTx struct {
+	ChainID              EthUint64   `json:"chainId"`
+	Nonce                EthUint64   `json:"nonce"`
+	Hash                 EthHash     `json:"hash"`
+	BlockHash            EthHash     `json:"blockHash"`
+	BlockNumber          EthUint64   `json:"blockNumber"`
+	TransactionIndex     EthUint64   `json:"transacionIndex"`
+	From                 EthAddress  `json:"from"`
+	To                   *EthAddress `json:"to"`
+	Value                EthBigInt   `json:"value"`
+	Type                 EthUint64   `json:"type"`
+	Input                EthBytes    `json:"input"`
+	Gas                  EthUint64   `json:"gas"`
+	GasLimit             *EthUint64  `json:"gasLimit,omitempty"`
```

Comment:
I can't find this in the docs.

Reply:
@ychiaoli18 – do you recall why this was necessary? I wasn't able to find it in the docs either.

Reply:
It seems that this was included accidentally. I'll remove it.